nixos/modules/services/continuous-integration/gocd-server/default.nix

```diff
@@ -67,9 +67,18 @@ in {
         '';
       };
 
+      javaPackage = mkOption {
+        default = pkgs.adoptopenjdk-jre-hotspot-bin-15;
+        defaultText = "pkgs.adoptopenjdk-jre-hotspot-bin-15";
+        type = types.package;
+        description = ''
+          Java package run the Go.CD server's process.
+        '';
+      };
+
       packages = mkOption {
-        default = [ pkgs.stdenv pkgs.jre pkgs.git config.programs.ssh.package pkgs.nix ];
-        defaultText = "[ pkgs.stdenv pkgs.jre pkgs.git config.programs.ssh.package pkgs.nix ]";
+        default = [ pkgs.stdenv pkgs.git config.programs.ssh.package pkgs.nix ];
+        defaultText = "[ pkgs.stdenv pkgs.git config.programs.ssh.package pkgs.nix ]";
```

Comment:
Very minor nitpick: I believe these should be using `literalExample`, so that they can be rendered without quotes